.github/workflows/main.yml

```diff
@@ -21,7 +19,9 @@ jobs:
         key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
         restore-keys: |
           ${{ runner.os }}-gems-
-    - run: bundle config path vendor/bundle
```

Comment:
Why was this removed? This is part of the bundle caching setup isn't it?

Reply:
I was playing around with using the built-in [`bundler-cache: true` option](https://github.com/ruby/setup-ruby#caching-bundle-install-automatically) in the `ruby/setup-ruby` action, which would let us remove a bunch of code here. Unfortunately, when I enable that the `jruby` build stops working and I wasn't able to figure out why. (I did make https://github.com/ruby/setup-ruby/issues/72)